README.md

```diff
@@ -84,54 +84,54 @@ val metaplex = Metaplex(solanaConnection, solanaIdentityDriver, storageDriver)
 ```
 
 # Usage
-Once properly configured, that `Metaplex` instance can be used to access modules providing different sets of features. Currently, there is only one NFT module that can be accessed via the `nft` property. From that module, you will be able to find, create and update NFTs with more features to come.
+Once properly configured, that `Metaplex` instance can be used to access modules providing different sets of features. Currently, only one NFT module can be accessed via the `nft` property. From that module, you will be able to find, create and update NFTs with more features to come.
```

Comment:
change to "Currently, there is only one module available: the NFT module that can be that can be accessed via..."